{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/static/js/src/{{cookiecutter.package_name}}.js

```diff
@@ -17,6 +17,12 @@ function {{cookiecutter.class_name}}(runtime, element) {
     });
 
     $(function ($) {
+        /*
+        Use `gettext` provided by django-statici18n for static translations
+
+        var gettext = {{cookiecutter.class_name}}i18n.gettext;
+        /*
```

Comment:
@viadanna looks like a backwards `/*` here

Reply:
Yeah, let me correct that.